portfolio/src/main/webapp/style.css

```diff
@@ -0,0 +1,48 @@
+body{
+    margin:0;
+}
+
+
+#fullpage{
+    min-width: 100vw;
+    min-height: 100vh;
```

Comment:
[Indent by 2 spaces at a time.](https://google.github.io/styleguide/htmlcssguide.html#Indentation)

Reply:
Spacing reformatted in commit 474248a